samples/cellular/lwm2m_client/sample_description.rst

```diff
@@ -397,9 +407,140 @@ CONFIG_SENSOR_MODULE_LIGHT - Configuration for light reading
 CONFIG_SENSOR_MODULE_COLOR - Configuration for color
    This configuration option enables the reading of color values.
 
+.. _lwm2m_configuration_files:
+
+Configuration files
+===================
+
+The sample provides predefined configuration files for typical use cases.
+
+* :file:`prj.conf` - Default configuration file.
+
+LwM2M Device management server:
+
+* :file:`overlay-leshan-bootstrap.conf` - Enables LwM2M bootstrap support with Leshan demo server.
+* :file:`overlay-avsystem.conf` - Uses `Coiote Device Management`_ server.
+* :file:`overlay-avsystem-bootstrap.conf` - Uses Coiote in bootstrap mode.
+
+NB-IoT:
+
+* :file:`overlay-nbiot.conf` - Enables the use of NB-IoT.
+
+LwM2M v1.1:
+
+* :file:`overlay-lwm2m-1.1.conf` - Enables LwM2M v1.1 protocol version.
+
+Firmware update:
+
+* :file:`overlay-adv-firmware.conf` - Enables experimental Advanced Firmware Update object.
+* :file:`overlay-fota_helper.conf` - Enables faster response for evaluating FOTA.
+
+Location assintance:
+
+* :file:`overlay-assist-agnss.conf` - Enables A-GNSS assistance.
+* :file:`overlay-assist-cell.conf` - Enables cell-based location assistance.
+* :file:`overlay-assist-pgps.conf` - Enables P-GPS assistance in the sample.
+
+Location service requirerequire `Coiote Device Management`_ server and LwM2M v1.1.
+
+DTLS optimications:
+
+* :file:`overlay-dtls-cid.conf` - Enables DTLS Connection Identifier.
+
+Power savings:
+
+* :file:`overlay-aggressive-psm.con` - Enables optimized PSM setup and proprietary PSM mode.
+* :file:`overlay-lowpower.conf` - Disables serial console to bring the power consumption down.
+
+LwM2M v1.1 conformance testing:
+
+* :file:`overlay-lwm2m-1.1-core-interop.conf` - Allows running of Core Specific Objects Test cases.
+* :file:`overlay-lwm2m-1.1-object-interop.conf` - Allows running of Additional Objects Test cases.
+
+
 Additional configuration
 ========================
 
+Default configuration
+---------------------
+
+LwM2M configuration:
+
+* Protocol version: 1.0
+* Binding mode: Queue
+* Device Management server: Leshan Demo server
+* Security: Enabled with PSK and DTLS session caching
+* Registration life time: 12 hours
+* Coap ACK initial timeout: 4 seconds
+* Enable LwM2M tickless mode power optimization
+* LwM2M shell
+
+Modem configurations:
+
+* Network Mode: LTE-M with GNSS.
+* PSM: Enabled TAU 60 minutes, RAT 30 seconds
+* Paging window: LTE 1.28 seconds and NB-IoT 2.56 seconds
+* eDRX: Enabled, with request of 10.24 seconds on LTE and 20.48 seconds on NB-Iot.
+* TAU pre-warning enabled, notification triggers registration update and TAU will be sent with the update which decreases power consumption.
+
+DTLS Connection Identifier
+--------------------------
+
+Add :file:`overlay-dtls-cid.conf` for activate DTLS Connection Identifier.
+Connection Identifier is recommend enable because it will reduce DTLS reconnection data traffic and also solve network NAT
+timeout problems from client to server side.
```

Comment:
```suggestion
It is recommended to enable the Connection Identifier, because it reduces DTLS reconnection data traffic and also solves the network NAT timeout problems from client to server side.
```